pixi.toml

```diff
@@ -1,20 +1,41 @@
 [project]
-name = "gluex-nstar"
-version = "0.1.0"
-description = "Add a short description here"
 authors = ["Vitor Shen <17490173+shenvitor@users.noreply.github.com>"]
 channels = ["conda-forge"]
+description = "Add a short description here"
+name = "gluex-nstar"
 platforms = ["linux-64"]
+version = "0.1.0"
 
 [tasks]
+docnb = """
+sphinx-build \
+    -nW --keep-going \
+    -b html \
+    docs _build/html
+"""
+docnblive = """
+sphinx-autobuild \
+    --watch docs \
+    -nW --keep-going \
+    -b html \
+    docs _build/html
+"""
 
 [dependencies]
-jaxlib = ">=0.4.23,<0.5"
-sympy = ">=1.12,<2"
-python = ">=3.12.3,<3.13"
-jupyterlab = ">=4.2.0,<4.3"
-jax = ">=0.4.26,<0.5"
-numpy = ">=1.26.4,<1.27"
 black = ">=24.4.2,<24.5"
 isort = ">=5.13.2,<5.14"
+jax = ">=0.4.26,<0.5"
+jaxlib = ">=0.4.23,<0.5"
+jupyterlab = ">=4.2.0,<4.3"
 jupyterlab-git = ">=0.50.0,<0.51"
+jupyterlab-lsp = ">=5.1.0,<5.2"
+jupyterlab-myst = ">=2.4.2,<2.5"
+myst-nb = ">=1.1.0,<1.2"
+nodejs = ">=20.12.2,<20.13"
```

Comment:
What is `nodejs` required for? I think you can just remove it (`pixi remove nodejs`)

Reply:
Removed :)